pom.xml

```diff
@@ -68,6 +68,12 @@
                 <enabled>true</enabled>
             </snapshots>
         </repository>
+
+<!--        Placeholder for local import of an unpublished library to maven repo. (i.e this is a local repository)-->
+        <repository>
+            <id>local-maven-repo</id>
+            <url>file:///${project.basedir}/local-maven-repo</url>
```

Comment:
What about pushing a version of the sdk to nexus so we don't have to commit the binary to the repo?

Reply:
Yeah, thats how I would like it too. 
Are we talking about a nexus repo internally(snowflake.int..) or onto public nexus?

I think there would be problems with accessing internal repo from github actions. 

Fetching it from public maven would be great. For that we would have to do a beta release of our sdk if we are ready. 

CC: @sfc-gh-tzhang

Reply:
i think we're ready, let's do it! Is there a way to prevent random customers from using the beta release?

Reply:
We can definitely do it from our internal nexus repo, I'll defer to @sfc-gh-tzhang and @sfc-gh-tjones on whether we're ready to push a beta to public maven

Reply:
@sfc-gh-mnaides I am fine keeping it as is. And once it is released, I will remove jar and fetch it from maven. 

@sfc-gh-tzhang - Umm, we will have beta name in our release which itself will prevent customers from using it IMO.